build.gradle

```diff
@@ -18,9 +18,24 @@ ext['snakeyaml.version'] = '1.32'
 
 group = 'uk.gov.hmcts.reform'
 
-allprojects {
-  sourceCompatibility = '11'
-  targetCompatibility = '11'
+java {
+  toolchain {
+    languageVersion = JavaLanguageVersion.of(17)
+  }
+}
+
+configurations.configureEach {
+  resolutionStrategy {
+    eachDependency { details ->
+      if (details.requested.group == 'org.projectlombok' && details.requested.name == 'lombok') {
+        details.useVersion '1.18.28'
+      }
+    }
+  }
```

Comment:
seems a bit odd to do this, why not just use lombok gradle plugin?

Reply:
removed as version was already specified elsewhere